MC/CustomGenerators/PWGDQ/Hijing_Jpsiee001.C

```diff
@@ -4,8 +4,9 @@ GeneratorCustom()
 
   TString simulation = gSystem->Getenv("CONFIG_SIMULATION");
     
-  AliGenCocktail *ctl = GeneratorCocktail("Hijing_Jpsiee001");
-
+  AliGenerator   *gen =  GeneratorCocktail("Hijing_Jpsiee001");
+  AliGenCocktail *ctl = (AliGenCocktail*) gen;
```

Comment:
Why not:
AliGenCocktail *ctl = (AliGenCocktail*)GeneratorCocktail("Hijing_Jpsiee001");
and then you return ctl?
This should work

Reply:
I thought since the macro returns AliGenerator it should return this. I tested and indeed it works just casting.